Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -365,6 +382,11 @@ void MediaPlayerPrivateGStreamerMSE::updateStates()
             GST_ERROR_OBJECT(pipeline(), "Setting the pipeline to PAUSED failed");
         m_isPipelinePlaying = false;
     }
+
+    if (getStateResult == GST_STATE_CHANGE_SUCCESS && state >= GST_STATE_PAUSED) {
+        // Update playback rate that was set before the pipeline was prerolled.
+        updatePlaybackRate();
+    }
```

Comment:
What we see here is that we need to prevent this, at least, if `m_isPlaybackRatePaused` is true, because it would force playback again. Can you explain a bit more?

Reply:
To be honest, this part is picked from 2.28 and I'm also not sure if needed really as I was not able to trigger this case. updatePlaybackRate() exits on very first check for m_isChangingRate (it is false). If you find this problematic we can just remove this part.

Reply:
@calvaris the following commit should fix the problem you mention: https://github.com/rawoul/WebKit/commit/e8ed3860c36a287fea835aaca954ec0eaf6358a5